.junie/guidelines.md

```
@@ -0,0 +1,321 @@
+   - @tailwind base;
+   - @tailwind components;
+   - @tailwind utilities;
+   + @import "tailwindcss";
+</code-snippet>
+
+
+### Replaced Utilities
+- Tailwind v4 removed deprecated utilities. Do not use the deprecated option - use the replacement.
+- Opacity values are still numeric.
+
+| Deprecated |	Replacement |
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 15

(MD010, no-hard-tabs)